Bottom: e28d49938a715bc97566af617560e7bf908d0f24
Top:    13df4f970ee8003bf7bfde3dede14e81508ed8bd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 22:04:36 +0000

Undo the perspective stuff


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 984a0ca..edb7498 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1659,8 +1659,6 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   Settings.ShowSolderSide = save_show_solder;
 }
 
-static int one_shot = 1;
-
 #define Z_NEAR 3.0
 gboolean
 ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
@@ -1712,26 +1710,13 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
 
   glMatrixMode (GL_PROJECTION);
   glLoadIdentity ();
-//  glOrtho (0, widget->allocation.width, widget->allocation.height, 0, -100000, 100000);
-//  glOrtho (-1, 1, -1, 1, 100, 10000);
-//  glFrustum (0, widget->allocation.width, widget->allocation.height, 0, 10, 1000);
-  glFrustum (-1, 1, -1, 1, 10, 10000);
-//  gluPerspective (30, widget->allocation.width / widget->allocation.height, 10, 1000);
+  glOrtho (0, widget->allocation.width, widget->allocation.height, 0, -100000, 100000);
   glMatrixMode (GL_MODELVIEW);
   glLoadIdentity ();
 
-  glScalef (2, 2, 1);
-
-  glTranslatef (-1., 1., 0.);
-  glScalef ( 2. / (float)widget->allocation.width,
-            -2. / (float)widget->allocation.height,
-             1. / 100.);
-
-  glTranslatef (0., 0., -2000.);
   glTranslatef (widget->allocation.width / 2., widget->allocation.height / 2., 0);
   glMultMatrixf ((GLfloat *)view_matrix);
   glTranslatef (-widget->allocation.width / 2., -widget->allocation.height / 2., 0);
-  glScalef (1., 1., 1. / 3.);
   glGetFloatv (GL_MODELVIEW_MATRIX, (GLfloat *)last_modelview_matrix);
 
 #if 0
